webpack.mix.js

```diff
@@ -1,7 +1,7 @@
 const mix = require("laravel-mix");
 const sass = require("node-sass");
 const cssnano = require("cssnano");
-const autoprefixer = require("autoprefixer");
+// const autoprefixer = require("autoprefixer");
```

Comment:
Any reason the autoprefixer was commented out?

Reply:
I don't think it's being used. But I didn't mean to leave it like that. Undone.